packages/forms/src/UIForm/fields/Button/Buttons.scss

```diff
@@ -1,14 +1,21 @@
 .tf-buttons {
 	display: flex;
 	justify-content: space-between;
+	width: 100%;
 
 	.left {
-		margin-left: $padding-large;
 		margin-right: auto;
+
+		+ .left {
+			margin-right: $padding-large;
+		}
```

Comment:
I don't get this part. The following `right` makes sense
```
(right_btn)     _margin_(right_btn)    _margin_(right_btn)
```

But this left part seems weird
```
(left_btn)    (left_btn)_margin_    (left_btn)_margin_
```